app/views/content/steps-to-become-a-teacher/_step_5_apply_for_teacher_training.html.erb

```diff
@@ -4,4 +4,9 @@
 
 <p><a href="/how-to-apply-for-teacher-training">Get tips on making a great application</a> including finding the right references and writing a personal statement.</p>
 
+<%= render Content::ExpanderComponent.new(
+    title: "applying for training and visas",
+    text: "If you need a visa to train to teach in England, you must have a confirmed offer of a training place before you apply for your visa. You should only apply for courses that support student or skilled worker visa sponsorship.",
```

Comment:
@emmaachesongray do we need to add signposting to the visas for trainees page here? probably a question for Amy